mdoc/Mono.Documentation/Updater/MsxdocDocumentationImporter.cs

```diff
@@ -75,8 +83,12 @@ public override void ImportDocumentation (DocsNodeInfo info)
                     case "returns":
                         {
                             XmlElement v = e.OwnerDocument.CreateElement (info.ReturnIsReturn ? "returns" : "value");
-                            v.InnerXml = child.InnerXml;
-                            e.AppendChild (v);
+                            XmlElement p2 = (XmlElement)e.SelectSingleNode(child.Name);
+                            if (p2 == null)
```

Comment:
Unless I'm reading the code wrong (a distinct possibility :) ), should this `case` also contain a call to `NeedsOverwrite`?

Reply:
In this case p2 is already null, because we have cleared this node here:
 ```
 if (DocUtils.NeedsOverwrite(e["value"]))
    MDocUpdater.ClearElement(e, "value");
```
At the moment, `NeedsOverwrite` returns false for null values, so call `NeedsOverwrite` here will break the logic.

Reply:
Oh, I see ... carry on :)